base/time_samples.cpp

```diff
@@ -0,0 +1,24 @@
+#include "base/time_samples.hpp"
+
+#include "std/cmath.hpp"
+
+namespace my
+{
+void TimeSamples::Add(double s)
+{
+  m_s += s;
+  m_s2 += s * s;
+  ++m_total;
+}
+
+double TimeSamples::GetMean() const { return m_total == 0 ? 0.0 : m_s / m_total; }
+
+double TimeSamples::GetSd() const
```

Comment:
https://stat.ethz.ch/R-manual/R-devel/library/stats/html/sd.html

Where it's more common and recognizable? The R community disagree with you.